UI-Plan.md

```
@@ -0,0 +1,375 @@
+- Storage
+
+Others feel more like free-form overlays on the vault canvas:
+
+- Overview
+- Exploration
+
+For example, Storage has a strong centered column with large glowing panels, while Exploration has a lot of empty black space and a single small card on the left.
+
+**Suggestion:** define one reusable page shell:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)